eng/common/scripts/modules/Package-Properties/Package-Properties.psm1

```diff
@@ -0,0 +1,172 @@
+# Helper functions for retireving useful information from azure-sdk-for-* repo
+# Example Use : Import-Module .\eng\common\scripts\modules\Package-Properties\Package-Properties.psm1 -ArgumentList 'JavaScript','C:\Git\azure-sdk-for-js'
+Param
+(
+    [Parameter(Position=0, Mandatory=$true)]
+    [ValidateSet("net","java","js","python")]
+    [string]
+    $Language,
+    [Parameter(Position=1, Mandatory=$true)]
+    [string]
+    $RepoRoot
```

Comment:
I think you can default the reporoot location based on our eng/common convention.